src/transformers/models/reformer/modeling_reformer.py

```diff
@@ -2310,12 +2325,7 @@ def set_output_embeddings(self, new_embeddings):
         self.lm_head.decoder = new_embeddings
 
     @add_start_docstrings_to_model_forward(REFORMER_INPUTS_DOCSTRING)
-    @add_code_sample_docstrings(
-        processor_class=_TOKENIZER_FOR_DOC,
-        checkpoint=_CHECKPOINT_FOR_DOC,
-        output_type=MaskedLMOutput,
-        config_class=_CONFIG_FOR_DOC,
-    )
+    @replace_return_docstrings(output_type=MaskedLMOutput, config_class=_CONFIG_FOR_DOC)
```

Comment:
The reason for using `replace_return_docstrings` here is below:
- ReformerTokenizer's mask_token is `None`, so we need to set it. (line: 2360)
- `len(tokenizer.tokenize("The capital of France is [MASK]."))` is 16, but `len(tokenizer.tokenize("The capital of France is Paris."))` is 17. So, we need to match the lengths. (line: 2378)

Reply:
OK, Reformer is in fact an auto-regressive generative model, not really intended to be used as Masked LM model.
I saw that you already use `hf-internal-testing/tiny-random-reformer` for `ReformerForMaskedLM`, so we don't have issue regarding `is_decoder`.

We still have issue regarding `mask token`, so good to use `replace_return_docstrings`! Thank you.